ee/tables/execparsers/socketfilterfw/parser.go

```diff
@@ -0,0 +1,84 @@
+package socketfilterfw
+
+import (
+	"bufio"
+	"io"
+	"regexp"
+	"strings"
+)
+
+var lineRegex = regexp.MustCompile("(state|block|built-in|downloaded|stealth|log mode|log option)(?:.*\\s)([0-9a-z]+)")
+
+// socketfilterfw returns lines for each `get` argument supplied.
+// The output data is in the same order as the supplied arguments.
+//
+// Each line describes a part of the feature and what state it's in.
+// These are not very well-formed, so I'm doing some regex magic to
+// know which option the current line is, and then sanitize the state.
+func socketfilterfwParse(reader io.Reader) (any, error) {
+	results := make([]map[string]string, 0)
+	row := make(map[string]string)
```

Comment:
I'd recommend putting `row` into the `for scanner.Scan()` loop.

Outside the loop it optimizes reducing memory allocations, which can be a small win for high performance, but comes with the risk/overhead of needing to make sure you've cleared state each time. Since we're not high performance, it's simpler to put it into the loop and let the scoping handle it.

Reply:
Sounds good

Reply:
I updated how row is being handled since I added in apps, but I still have it first declared outside of the loop.

Reply:
Oh, I see now. 

In the general case, you're returning a single row, with a lot of columns. 

I was thinking you could return this as EAV style data, and let data flatten handle it.